app/assets/stylesheets/layout.scss

```diff
@@ -31,6 +31,10 @@
   font-weight: bold;
 }
 
+.normal-weight {
```

Comment:
normal-weight? surely this `bold`?

Reply:
I believe we already have bold classes too

Reply:
not used ... probably I forget to remove it